docs/source/settings.rst

```diff
@@ -56,6 +56,26 @@ If ``True``, register or activation endpoint will send confirmation email to use
 
 **Default**: ``False``
 
+SEND_REREGISTRATION_EMAIL
+-------------------------
+
+If ``True``, register endpoint will send warning and reminder email to user. 
+Only active users are able to reset their passwords (User.is_active is True).
+The email will ask the User to reset their password since someone is trying to
+re-register their account.
+If User.is_active is False, will send a warning
```

Comment:
Some of code values are decorated with higlight like \`\`True\`\`, but it seems that you have forgot about the others.